RELEASES.md

```diff
@@ -42,23 +46,28 @@ schedule, initially aiming for the bi-weekly releases.
 
 ### Release management
 
-Release managers of stable releases are responsible for approving and merging backports, tagging
-stable releases and sending announcements about them. This role is rotating on a quarterly basis.
-
-| Quarter |       Release manager                                          |
-|:-------:|:--------------------------------------------------------------:|
-| 2020 Q1 | Piotr Sikora ([PiotrSikora](https://github.com/PiotrSikora))   |
-| 2020 Q2 | Piotr Sikora ([PiotrSikora](https://github.com/PiotrSikora))   |
-| 2020 Q3 | Yuchen Dai ([lambdai](https://github.com/lambdai))             |
-| 2020 Q4 | Christoph Pakulski ([cpakulski](https://github.com/cpakulski)) |
-| 2021 Q1 | Rei Shimizu ([Shikugawa](https://github.com/Shikugawa))        |
-| 2021 Q2 | Dmitri Dolguikh ([dmitri-d](https://github.com/dmitri-d))      |
-| 2021 Q3 | Takeshi Yoneda ([mathetake](https://github.com/mathetake))     |
-| 2021 Q4 | Otto van der Schaaf ([oschaaf](https://github.com/oschaaf))    |
-| 2022 Q1 | Otto van der Schaaf ([oschaaf](https://github.com/oschaaf))    |
-| 2022 Q2 | Pradeep Rao ([pradeepcrao](https://github.com/pradeepcrao))    |
-
-## Release schedule
+Major releases are handled by the maintainer on-call and do not involve any backports.
+The details are outlined in the "Cutting a major release" section below.
+Security releases are handled by a Release Manager and a Fix Lead. The Release Manager is
+responsible for approving and merging backports. The Fix Lead is a member of the security
+team and is responsible for coordinating the overall release. This includes identifying
+issues to be fixed in the release, communications with the Envoy community, and the
+actual mechanics of the release itself.
```

Comment:
Hm... I might be biased here, but I strongly think we should document this. When I got signed up for the role I had a hard time understand what it was and how it related to other parts of the process. In addition, I was recently fielding questions about the division of labor between the "release manager" and the "fix lead" and the existing documentation did really seem to make this clear. I think it would be quite helpful to at least have this written down. I also tend to think that as Envoy increases in size and visibility it'll be increasingly likely that we will have security bug every quarter. But I could totally be wrong!